backend/sublet/serializers.py

```diff
@@ -122,7 +122,9 @@ def destroy(self, instance):
 
 
 class SubletSerializerRead(serializers.ModelSerializer):
-    amenities = AmenitySerializer(many=True, required=False)
+    amenities = serializers.PrimaryKeyRelatedField(
```

Comment:
Thanks! Can we confirm this change by adding an additional check in one of our test cases? Should just be one additional `assert` or something to assert that the type now returned is a list of strings

Reply:
Fixed! https://github.com/pennlabs/penn-mobile/pull/251/commits/19a351c49fd87dedcaf5fecc4aa058dcbf72d7f8 checks this by a direct comparison to the expected string list output.